xds/src/test/java/io/grpc/xds/XdsLoadBalancerProviderTest.java

```diff
@@ -73,92 +73,30 @@ public String getPolicyName() {
     public LoadBalancer newLoadBalancer(Helper helper) {
       return fakeBalancer1;
     }
-  };
-
-  private final LoadBalancerProvider roundRobin = new LoadBalancerProvider() {
-    @Override
-    public boolean isAvailable() {
-      return true;
-    }
-
-    @Override
-    public int getPriority() {
-      return 5;
-    }
-
-    @Override
-    public String getPolicyName() {
-      return "round_robin";
-    }
 
     @Override
-    public LoadBalancer newLoadBalancer(Helper helper) {
-      return null;
+    public ConfigOrError parseLoadBalancingPolicyConfig(
+        Map<String, ?> rawLoadBalancingPolicyConfig) {
+      return ConfigOrError.fromConfig(lbConfig1);
     }
   };
 
+  private final LoadBalancerProvider roundRobinProvider =
```

Comment:
Wait, why do you do this? I mean you could just use the default registry instance for test.

Reply:
Because I didn't like to modify the global registry.